sandbox.sh

```diff
@@ -69,6 +73,7 @@ EOF
   echo "]"
 }
 
+# Create a json file containing the trusted validators (subset of all validators.json created before)
```

Comment:
This comment is misleading. It is not required that trusted-validator-membership-change.json be a subset of validators.json. If this were the case, how would new validators join the network at a later time?

Reply:
Thank you. So it is only the "starting list" of trusted validators? Which could be only the `validator0`?
If yes, I think we could use distinct variables for `validators=(0 1 2)` and `trusted_validators=(0 1 2)` and explain it. What do you think?

Reply:
I'm not sure it's worth it to represent the full complexity of what's possible in Deku in the sandbox.sh script. It is provided merely as a convenience for more efficient hacking on Deku. I would defer implementing a separation between validators and trusted validator changes until you need it for manual testing.